src/libgit2/repository.h

```diff
@@ -102,6 +102,8 @@ typedef enum {
 	/* core.trustctime */
 	GIT_TRUSTCTIME_DEFAULT = GIT_CONFIGMAP_TRUE,
 	/* core.abbrev */
+	GIT_ABBREV_FALSE = GIT_CONFIGMAP_FALSE,
```

Comment:
```suggestion
	GIT_ABBREV_FALSE = INT_MAX,
```

This could be `GIT_OID_MAX_HEXSIZE` which might be more obvious to the reader?